src/agent/rustjail/src/container.rs

```diff
@@ -1554,19 +1554,35 @@ mod tests {
     use std::os::unix::fs::MetadataExt;
     use std::os::unix::io::AsRawFd;
     use tempfile::tempdir;
+    use tokio::process::Command;
 
     macro_rules! sl {
         () => {
             slog_scope::logger()
         };
     }
 
+    async fn which(cmd: &str) -> String {
+        let output: std::process::Output = Command::new("which")
```

Comment:
I wonder if we should be using https://crates.io/crates/which to avoid relying on an external command.